src/FakerDotNet/Algorithms/VinCheckSum.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+namespace FakerDotNet.Algorithms
```

Comment:
I've dropped this into the namespace, but we can move it, it doesn't feel quite right, as it's not an algorithm.

Reply:
Maybe there could be a Checksum namespace? I think there may be a few areas where we need to generate checksums, so having them all grouped together in the same namespace would be beneficial 🤔

Also, should the class be called VinChecksum?